content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
+The above does not mean that if the quizzer needs to provide a reference question that is a CVR, and their choice of interrogative word changes the question type to MACVR, that the quizzer has given a valid CVR. In this case, the validity changes based on the interrogative word because of how the required answer changes.
 
-##### Chapter-Verse (CVR)
+##### Chapter Verse Reference (CVR)
 
-- Must have a single answer.
-- The question must contain a word or phrase that exists in more than one verse in the same chapter.
-- Context is the single verse.
+- Must have a single answer
+- The question must contain a word or phrase that exists in more than 1 verse in the same chapter
+- Context is the single verse
 
-##### Multiple-Answer, Chapter-Verse (MACVR)
+The question text should be in the form:
 
-- Must meet all the requirements of a CVR except the single answer requirement.
-- Must also meet all the requirements of a MA except the unique word/phrase requirement.
+> According to [book name], chapter [chapter number], verse [verse number], [remainder of question text]
 
-##### Chapter (only) (CR)
+##### Chapter Reference (CR)
 
-The question must contain a word or phrase that exists more than once in the material and only once in the chapter.
+- Must have a single answer
+- Must contain a word or phrase that exists more than once in the material and only once in the chapter
 
-##### Multiple-Answer, Chapter (only) (MACR)
+The question text should be in the form:
 
-- Must meet all the requirements of a CR except the single answer requirement.
-- Must also meet all the requirements of a MA except the unique word/phrase requirement.
+> According to [book name], chapter [chapter number], [remainder of question text]
+
+##### Multiple Answer Chapter Verse Reference (MACVR)
+
+- Must meet all the requirements of a CVR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
+
+##### Multiple Answer Chapter Reference (MACR)
+
+- Must meet all the requirements of a CR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
 
 #### Quote/Finish
 
-- Must be quoted word-perfect.
+Quote and Finish questions must be quoted word-perfect. Quizzers may go back and correct their answers if done in the question answer duration time period. The quizzer will be required to make 1 full correct rotation; however, the quizzer can be called correct if they have made a mistake only in 1 area of the verse and correct the mistake, making it abundantly clear to the quizmaster and answer judge that they know where their mistake was and what the corrected words are.
```

Comment:
The "however" does the work, but I think it can be made more clear "however, the quizzer can be called correct WITHOUT 1 full correct rotation if...."

Reply:
861185c